ui/app/AppLayouts/Wallet/popups/simpleSend/SimpleSendModal.qml

```diff
@@ -340,6 +334,17 @@ StatusDialog {
             }
         }
 
+        function setSelectedAddress() {
+            if (!!root.selectedRecipientAddress || !root.selectedAddress)
+                return
+
+            if (Utils.isValidEns(root.selectedAddress)) {
+                recipientsPanel.setText(root.selectedAddress)
```

Comment:
maybe you should use root.ensName here instead?

Reply:
ok got it, then maybe you can remove the ensName property from this modal? and the fillinf it part in the handler?